### ISSUE TITLE
Problems encountered during visualization of results

### ISSUE THREAD

**Issue by lxin98** [spla-tam/SplaTAM#132]
Thank you for your work. I am running Python viz_stcripts/final_decon.exe configs/realsense/splatam.py
When coding, there was an error called 'Process finished with exit code 139' (interrupted by signal 11: SIGSEGV). I debugged and found that it was at line 202 where frustum=o3d. geometry LineSet.creat_camera_visualization (w, h, k, allw_2cs [i_t], frustumsize) encountered an error. How can I resolve it

**Comment by lxin98** [comment]
![3599385773](https://github.com/user-attachments/assets/f9eaec1e-0fea-4a66-bd84-8468c3354ce5)

**Comment by Nik-V9** [comment]
Hi, Thanks for trying out our code!

Do you have the right version  of Open3D installed?

https://github.com/spla-tam/SplaTAM/blob/da6bbcd24c248dc884ac7f49d62e91b841b26ccc/requirements.txt#L11

**Comment by lxin98** [comment]
yes ，I installed the right version0.16.0，I am going to create a new conda enviroment。I plan to create a new conda environment to see if it can run @Nik-V9 ，thank you for you help
